lib/jxl/dec_ans.h

```diff
@@ -235,7 +235,11 @@ class ANSSymbolReader {
     return ReadSymbolWithoutRefill(histo_idx, br);
   }
 
-  bool CheckANSFinalState() { return state_ == (ANS_SIGNATURE << 16u); }
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
`#ifndef ... #else` is a bit surprising, I read it the wrong way around the first time. I wonder if it wouldn’t be clearer with `#ifdef`, but then maybe it’s just me.